concent_api/api-integration-force-accept-or-reject-results-test.py

```diff
@@ -101,40 +94,87 @@ def report_computed_task(timestamp = None, subtask_id = None, task_to_compute =
         )
 
 
+def create_task(task_to_compute_timestamp, task_id, compute_task_def_deadline):
+    return task_to_compute(
+        timestamp=task_to_compute_timestamp,
+        compute_task_def=compute_task_def(
+            task_id=task_id,
+            subtask_id=task_id,
+            deadline=compute_task_def_deadline,
+        )
+    )
+
+
+ProtocolConstants = namedtuple("ProtocolConstants",
+                               ["concent_messaging_time",
+                                "subtask_verification_time",
+                                "force_acceptance_time",
+                                "token_expiration_time"])
+
+
+def get_protocol_constants(cluster_url):
+    url = f"{cluster_url}/api/v1/protocol-constants/"
+    resp = requests.get(url)
+    json = resp.json()
+    concent_messaging_time = json['concent_messaging_time']['value']
+    subtask_verification_time = json['subtask_verification_time']['value']
+    force_acceptance_time = json['force_acceptance_time']['value']
+    token_expiration_time = json['token_expiration_time']['value']
+    constants = ProtocolConstants(concent_messaging_time, subtask_verification_time, force_acceptance_time,
+                                  token_expiration_time)
+    return constants
+
+
+def print_protocol_constants(constants):
+    print("PROTOCOL_CONSTANTS: ")
+    print(f"concent_messaging_time = {constants.concent_messaging_time}")
+    print(f"subtask_verification_time = {constants.subtask_verification_time}")
+    print(f"force_acceptance_time = {constants.force_acceptance_time}")
+    print(f"token_expiration_time = {constants.token_expiration_time}\n")
+
+
 def main():
     cluster_url     = parse_command_line(sys.argv)
     task_id         = str(random.randrange(1, 100000))
     current_time    = get_current_utc_timestamp()
 
+    cluster_consts = get_protocol_constants(cluster_url)
+    print_protocol_constants(cluster_consts)
+
+    current_timestamp = timestamp_to_isoformat(current_time)
+    ack_report_computed_task_timestamp = timestamp_to_isoformat(
```

Comment:
We don't use hard line length limit, but suggested is 120, so it is ok to put it in single line.